### ISSUE TITLE
Calamari incorrectly reporting version 0.0.0 in deployment task logs

### ISSUE THREAD

**Issue by michaelnoonan** [OctopusDeploy/Issues#3018]
When deploying a release of a project, the Calamari version is logged as `0.0.0` even though it should be something like `3.4.5`.

https://demo.octopus.com/app#/tasks/ServerTasks-390083/raw

```
                    |   == Success: Step 1: Say hello ==
00:12:50   Verbose  |     Say hello completed
                    |   
                    |     Success: Octopus Server
00:12:46   Verbose  |       Octopus Server version: 3.7.6+Branch.master.Sha.7e2f2b8bca051785b194401a592746138b243708
00:12:46   Verbose  |       Environment Information:
                    |       OperatingSystem: Microsoft Windows NT 6.2.9200.0
                    |       OsBitVersion: x64
                    |       Is64BitProcess: True
                    |       CurrentUser: NT AUTHORITY\SYSTEM
                    |       MachineName: DEMO
                    |       ProcessorCount: 4
                    |       CurrentDirectory: C:\Windows\system32
                    |       TempDirectory: C:\Windows\TEMP\
                    |       HostProcessName: Octopus.Server
00:12:46   Verbose  |       Octopus Deploy: Calamari version 0.0.0
```

**Comment by michaelnoonan** [comment]
This has been fixed in Calamari `3.6.9`

![image](https://cloud.githubusercontent.com/assets/1627582/21252878/27baa5bc-c3a6-11e6-9061-4986fe62c48d.png)

**Comment by michaelnoonan** [comment]
Release Note: Calamari is now versioned correctly and reports the correct version number in the Task Logs
